.github/workflows/release-build.yml

```diff
@@ -98,18 +98,12 @@ jobs:
     name: Build ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.cpack_gen }} Installer
     strategy:
       matrix:
-        id: [windows-x64-nsis, windows-x64-zip, macos-x64-zip]
+        id: [windows-x64, macos-x64-zip]
         include:
-          - id: windows-x64-nsis
+          - id: windows-x64
             os: windows-latest
             arch: x64
-            cpack_gen: NSIS
-            cmake_gen: 'Visual Studio 16 2019'
-            msvc_ver: 'msvc2019'
-          - id: windows-x64-zip
-            os: windows-latest
-            arch: x64
-            cpack_gen: ZIP
+            cpack_gen: NSIS;ZIP
```

Comment:
cpack can do more than one at once? or is this used by a script some where?

Reply:
CPack takes a list for the generator(s) to use -- the default on Linux (if not set by an argument or in the CMake project) is creating tgz, stgz, and (tbz2?) archives when cpack is run.